### PR TITLE
Use a text field for license in pyproject.toml

### PR DESCRIPTION
**Pull request by leouieda**
When using the "file" field type, PyPI putting the entire contents of the file into the license field and mislabeling it as MIT. Replace it with just the string for the license name and not the contents of the file.

### CODE REVIEW

**Comment by leouieda** [comment]
See https://test.pypi.org/project/harmonica/ for the incorrect output.

**Comment by santisoler** [comment]
Nice catch @leouieda!

**Comment by leouieda** [comment]
Ah no wait, there is also the classifier causing problems.

**Comment by leouieda** [comment]
Can't fix it now, gotta run

**Comment by leouieda** [comment]
Alright, done! Ready for a review. I made a few tweaks as well to put fields closer to other relevant fields.

File: pyproject.toml [hunk 1]
```diff
@@ -1,26 +1,22 @@
...
 ]
 readme = "README.md"
-requires-python = ">=3.8"
-license = {file = "LICENSE.txt"}
+license = {text = "BSD-3-Clause"}
 keywords = ["geoscience", "geophysics"]
 classifiers = [
     "Programming Language :: Python :: 3",
-    "License :: OSI Approved :: MIT License",
+    "License :: OSI Approved :: BSD License",
```
Comment (santisoler) [inline pyproject.toml:16]: Ooops. Thanks for the fix!

**Comment by santisoler** [comment]
I'm merging this. Thanks @leouieda!

**Pull request merged**